pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
 
   <artifactId>p4</artifactId>
-  <version>1.7.6-SNAPSHOT</version>
+  <version>1.7.7-SNAPSHOT-FGOL</version>
```

Comment:
Can you revert back to 1.7.6 or this will break the next release.

Reply:
Ooh sorry, it was originally for internal use :) 
One sec :)

Reply:
Removed version and author modifications.